Interface-Signoff-Marvo.md

```diff
@@ -0,0 +1,48 @@
+# Interface Signoff
+
+## Subsystem Function
+The interface conditions any input analog voltage, analog current and digital signal then transmits it to the microcontroller. 
+## Subsystem Constraints
+1) Must be able to operate with mV voltage output ranges
+2) Must support positive DC voltage ranges: 0 to 1 V, 0 to 5 V, 1 to 5 V, 0 to 10V
+3) Must be able to operate with negative voltage ranges: -10 to 10 V
+4) Must Support current output range: 4-20 mA
+5) The PCB material used must be UL certified
+
+
+## System Schematic
+
```

Comment:
What purpose does the rectifier serve?

Reply:
rectifier has been taken out and voltage ranges have been instead scaled accordingly